#
#

Header header
uint32 action   # 0 : No action; 1 : Close; 2 : Open;
uint32 cargoID  # 1 : Front cargo; 2 : Rear cargo;
